instrumentation/rxjava-2.0/javaagent/src/main/java/io/opentelemetry/instrumentation/rxjava2/RxJava2InstrumentationModule.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.rxjava2;
+
+import static net.bytebuddy.matcher.ElementMatchers.isTypeInitializer;
+import static net.bytebuddy.matcher.ElementMatchers.named;
+
+import io.opentelemetry.javaagent.tooling.InstrumentationModule;
+import io.opentelemetry.javaagent.tooling.TypeInstrumentation;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.description.type.TypeDescription;
+import net.bytebuddy.matcher.ElementMatcher;
+
+public class RxJava2InstrumentationModule extends InstrumentationModule {
+
+  public RxJava2InstrumentationModule() {
+    super("rxjava2");
+  }
+
+  @Override
+  public List<TypeInstrumentation> typeInstrumentations() {
+    return Collections.singletonList(new PluginInstrumentation());
+  }
+
+  public static class PluginInstrumentation implements TypeInstrumentation {
+
+    @Override
+    public ElementMatcher<? super TypeDescription> typeMatcher() {
+      return named("io.reactivex.plugins.RxJavaPlugins");
+    }
+
+    @Override
+    public Map<? extends ElementMatcher<? super MethodDescription>, String> transformers() {
+      return Collections.singletonMap(isTypeInitializer(), RxJavaPluginsAdvice.class.getName());
```

Comment:
This might sound weird, but can you hard-code the class name instead?
```suggestion
      return Collections.singletonMap(isTypeInitializer(), RxJava2InstrumentationModule.class.getName() + "$RxJavaPluginsAdvice");
```
[This doc](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/writing-instrumentation.md#why-hard-code-advice-class-names) from our contributing folder should explain why it is preferable to hard-code advice class names (although it does look a bit werid).